core/src/scss/components/secondary-nav/_secondary-nav.scss

```diff
@@ -0,0 +1,174 @@
+//
+// Secondary Navigation
+//
+// The secondary site navigation that is...
+//
+// .su-secondary-nav--no-interactivity - A plain option with no javascript interactivity.
+// .su-secondary-nav--light   - Light theme with white background. This variant is often used instead of the default.
+// .su-secondary-nav--buttons - Button toggles instead of link toggles.
+// .su-secondary-nav--buttons-light - Button toggles instead of link toggles, light edition.
+//
+// Markup: ../templates/components/secondary-nav/secondary-nav.twig
+//
+// Style guide: Components.Secondary Navigation
+//
+
+// Default styles for secondary nav core elements.
+.su-secondary-nav {
+  @include font-smoothing;
+  background-color: color(bg, $su-secondary-nav-colors);
+
+  // Expand everything when there is no javascript.
+  &.no-js {
+    .su-secondary-nav__menu {
+      display: block;
+    }
+  }
+}
+
+// Global styles for nested menu items.
+// ---------------------------------------------------------------------------
+.su-secondary-nav__menu {
+  @include margin(0 null);
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  list-style: none;
+  background-color: color(bg, $su-secondary-nav-colors);
+}
+
+// Global styles for list items.
+// ---------------------------------------------------------------------------
+.su-secondary-nav__item {
+  @include margin(null null 0);
+}
+
+// Global styles for secondary nav links.
+// ---------------------------------------------------------------------------
+.su-secondary-nav__link {
+  @include fancy-hover(
+    color(border--hover, $su-secondary-nav-colors),
+    color(border--active, $su-secondary-nav-colors),
+    'left'
+  );
+  @include padding(1.6rem 4.8rem 1.6rem 2.4rem);
+  display: block;
+  color: color(link, $su-secondary-nav-colors);
+  font-weight: $su-font-semi-bold;
+  font-size: 1.8rem;
+  border-top: color(divider, $su-secondary-nav-colors) solid 1px;
+  outline: 0;
+
+  &:focus,
+  &:hover {
+    color: color(link, $su-secondary-nav-colors);
+    text-decoration: underline;
+  }
+}
+
+// First level items
+// ---------------------------------------------------------------------------
+.su-secondary-nav__menu-lv1 {
+  > .su-secondary-nav__item {
+    .su-secondary-nav__link {
+      font-weight: $su-font-bold;
+    }
+  }
+}
+
+// Secondary Level Items
+// ---------------------------------------------------------------------------
+.su-secondary-nav__menu-lv2 {
+  .su-secondary-nav__item {
+    &:last-child {
+      @include padding(0.1rem null 0.6rem null);
```

Comment:
```suggestion
      @include padding(0.1rem null null null);
```

Reply:
I can't add it here properly with the formatting and I suspected this is not a real working branch, but the idea is that for the `<li>` last child, I'm adding the padding-bottom to the `<a>` element instead of the `<li>` so when you hover over the last link that's above a horizontal divider line, there is no space between the bottom of the thick left vertical bar and the horizontal divider line.